docs/02.API-REFERENCE.md

```diff
@@ -1998,7 +1998,8 @@ handler (const jerry_value_t function_obj,
 
   // after this, script can invoke the native handler through "handler_field (1, 2, 3);"
   jerry_value_t prop_name = jerry_create_string ((const jerry_char_t *) "handler_field");
-  jerry_set_property (glob_obj, prop_name, func_val);
+  jerry_value_t set_result = jerry_set_property (glob_obj, prop_name, func_val);
```

Comment:
@zherczeg That is true, however I wanted to emphasise that there is a returned value there. Also wanted to avoid multiple jerry api calls in one line to make the whole thing clearer. After all this is an API doc :)